templates/docs/examples/patterns/strips/dark.html

```diff
@@ -5,7 +5,7 @@
 
 {% set is_paper = true %}
 {% block content %}
-<div class="p-strip is-dark">
+<div class="p-strip">
```

Comment:
This example is used in the docs to show how to make strip dark, so in this case we want the `is-dark` class on strip, not on body (to show how to change single strip to dark).

So for now it's ok that theme switch will not work (or will not be available) on this example.